src/components/home/sections/About.js

```diff
@@ -13,7 +13,7 @@ export const About = () => {
     <section id='about'>
       <h2>{HEADER_TEXT}</h2>
       <div className='about-content-container'>
-        <img id='about-groceries' src={groceries} />
+        <div className='about-groceries-container'><img id='about-groceries' src={groceries} /></div>
```

Comment:
Interesting way of wrapping the image in a container. :)

Reply:
:) Sorry, I will separate them